<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NATS.io Explore</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="wtOptions">
       <attribute name="title">
        <string>Options</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="PathIniFile">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <italic>true</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>default.ini</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pbReloadini">
            <property name="text">
             <string>ReLoad ini</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wpbSaveini">
            <property name="text">
             <string>Save ini</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wpbOpenDialogini">
            <property name="text">
             <string>Open Dialog / Сменить ini</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Server NATS.io</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="weHostNameNATSio">
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Host Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Sockert Port</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="wvPortNATSio">
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>4222</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Account NATS.io user</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="QLineEdit" name="weUPasswd"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="weUName">
            <property name="text">
             <string>user</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>UPasswd</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>UName</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>ClientID</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="weClientId"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxNoSaveAccount">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>! НЕ Сохранять пароль и пользователя / dont save uname and upasswd</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wchbAutosaveExit">
          <property name="text">
           <string>Autosave before close / сохранять перед выходом</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wtExprore">
       <attribute name="title">
        <string>Explore</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="wrStatusMQ">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="wbConnect">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Connect NATS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="wbClose">
              <property name="text">
               <string>Disconnect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>ssid:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="inputMask">
               <string>99999999999999999999</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="wbUnScrible">
              <property name="text">
               <string>UnScrible</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QToolButton" name="wbAllTopics">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>topic</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="weTopicSub">
              <property name="text">
               <string>&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="wbSubscr">
              <property name="text">
               <string>Subscr</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Message Queue Explore log:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="sizeFornReser">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sizeFontUp">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sizeFontDn">
              <property name="text">
               <string>--</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="weOutputSubscr"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>topic</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="weTopicPub">
              <property name="text">
               <string>testTopic</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>payload</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="wePayload">
              <property name="text">
               <string>testPayload</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="wbPublic">
              <property name="text">
               <string>Public</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wtLogs">
       <attribute name="title">
        <string>Logs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTextEdit" name="weLogsOut">
          <property name="font">
           <font>
            <pointsize>6</pointsize>
            <italic>true</italic>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wtBiznesLogic">
       <attribute name="title">
        <string>BusinessLogic</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="wbSaveBizLogic">
            <property name="text">
             <string>Confirm, Save and Enable/on</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wbReLoad">
            <property name="text">
             <string>ReLoad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wbOffBizLog">
            <property name="text">
             <string>Disable/off</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="wtBusinessLogic"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
